### PR TITLE
Semantic release

### PR DESCRIPTION
**Pull request by thejamespower**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -51,13 +48,12 @@ jobs:
     name: Deploy Staging
     runs-on: ubuntu-20.04
     container:
-      image: docker/compose:1.29.2
+      image: timbru31/node-alpine-git
```
Comment (richardtbell) [inline .github/workflows/deploy.yml:51]: Something I was meaning to check, but I don't think we need both container and runs-on. If we use a container, it should have everything that's needed

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -16,9 +16,6 @@ on:
         required: true
         type: string
         default: master
-      BUILD_NUMBER:
```
Comment (richardtbell) [inline .github/workflows/deploy.yml:19]: Guess this can now be removed from the workflows calling it?

**Comment by richardtbell** [comment]
Would it be feasible to add the version to the slack notification?

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (thejamespower) [reply .github/workflows/deploy.yml:51]: `runs-on` is a required field :)

File: .github/workflows/notify-slack.yml [hunk 3]
```diff
@@ -32,7 +32,7 @@ jobs:
         with:
           script: |
             const commits = ${{ toJSON(github.event.commits) }};
-            return commits.map(c => c.message).join('\n');
+            return commits ? commits.map(c => c.message).join('\n') : 'Action was triggered by workflow dispatch';
```
Comment (richardtbell) [inline .github/workflows/notify-slack.yml:40]: Nice

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -99,7 +126,12 @@ jobs:
           $SLACK_NOTIFICATION_CI_RUNS
       - name: Set DOCKER_TAG
         run: |
-          echo "DOCKER_TAG=${REPOSITORY_URL}/emrit-${ENV}-${SERVICE}:${BUILD_NUMBER}" >> $GITHUB_ENV
+          if [[ $ENV == "prod" ]]
+          then
+            echo "DOCKER_TAG=${REPOSITORY_URL}/emrit-production-${SERVICE}:${TAG_TO_DEPLOY}" >> $GITHUB_ENV
```
Comment (richardtbell) [inline .github/workflows/deploy.yml:131]: Where does the discrepancy between prod and production occur?

**Review by richardtbell — APPROVED** [review]

File: .github/workflows/deploy.yml (hunk 4 above)
Comment (thejamespower) [reply .github/workflows/deploy.yml:131]: In AWS ECR for prod, services use the template `emrit-production-[service]`, but `ENV === 'prod'`

**Pull request merged**